code/modules/mob/living/basic/space_fauna/paper_wizard/paper_wizard.dm

```diff
@@ -0,0 +1,196 @@
+	var/list/empty_papers = list()
+
+	for(var/obj/item/paper/target_paper in oview(search_range, controller.pawn))
+		if(target_paper.is_empty())
+			empty_papers += target_paper
+
+	if(empty_papers.len)
+		return pick(empty_papers)
+
+/mob/living/basic/paper_wizard/copy
+	desc = "'Tis a ruse!"
+	health = 1
+	maxHealth = 1
+	alpha = 200
+	faction = list(FACTION_STICKMAN)
+	melee_damage_lower = 1
+	melee_damage_upper = 5
+
+	ai_controller = /datum/ai_controller/basic_controller/wizard_copy
+	///the master that summoned us
+	var/mob/living/basic/paper_wizard/original
+
+/mob/living/basic/paper_wizard/copy/grant_abilities()
+	return
+
+/mob/living/basic/paper_wizard/copy/grant_loot()
+	return
+
+//Hit a fake? eat pain!
+/mob/living/basic/paper_wizard/copy/death(gibbed)
+	. = ..()
+
+	for(var/mob/living/damaged in oview(5, src))
+		if(faction_check_mob(damaged, exact_match = FALSE))
+			continue
+		damaged.adjustBruteLoss(50)
+
+
+/mob/living/basic/paper_wizard/copy/examine(mob/user)
+	. = ..()
+	if(isobserver(user))
+		. += span_notice("It's an illusion - what is it hiding?")
+	else
+		qdel(src) //I see through your ruse!
+
+/datum/ai_controller/basic_controller/wizard_copy
+	blackboard = list(
+		BB_TARGETTING_DATUM = new /datum/targetting_datum/basic,
+	)
+
+	ai_movement = /datum/ai_movement/basic_avoidance
+	idle_behavior = /datum/idle_behavior/idle_random_walk
+	planning_subtrees = list(
+		/datum/ai_planning_subtree/simple_find_target,
+		/datum/ai_planning_subtree/basic_melee_attack_subtree,
+	)
+
+//fancy effects
+/obj/effect/temp_visual/paper_scatter
+	name = "scattering paper"
+	desc = "Pieces of paper scattering to the wind."
+	layer = ABOVE_NORMAL_TURF_LAYER
+	plane = GAME_PLANE
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "paper_scatter"
+	anchored = TRUE
+	duration = 5
+	randomdir = FALSE
+
+/obj/effect/temp_visual/paperwiz_dying
+	name = "craft portal"
+	desc = "A wormhole sucking the wizard into the void. Neat."
+	layer = ABOVE_NORMAL_TURF_LAYER
+	plane = GAME_PLANE
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "paperwiz_poof"
+	anchored = TRUE
+	duration = 18
+	randomdir = FALSE
+
+/obj/effect/temp_visual/paperwiz_dying/Initialize(mapload)
+	. = ..()
+	visible_message(span_boldannounce("The wizard cries out in pain as a gate appears behind him, sucking him in!"))
+	playsound(get_turf(src), 'sound/magic/mandswap.ogg', 50, vary = TRUE, pressure_affected = TRUE)
+	playsound(get_turf(src), 'sound/hallucinations/wail.ogg', 50, vary = TRUE, pressure_affected = TRUE)
+
+/obj/effect/temp_visual/paperwiz_dying/Destroy()
+	for(var/mob/nearby in range(7, src))
+		shake_camera(nearby, duration = 7 SECONDS, strength = 1)
+	var/turf/current_turf = get_turf(src)
+	playsound(current_turf,'sound/magic/summon_magic.ogg', 50, vary = TRUE, vary = TRUE)
+	new /obj/effect/temp_visual/paper_scatter(current_turf)
+	new /obj/item/clothing/suit/wizrobe/paper(current_turf)
+	new /obj/item/clothing/head/collectable/paper(current_turf)
+	return ..()
```

Comment:
I am iffy about `Initialize` having those side effects but `Destroy` _definitely_ should not have these sound effects.
Run a separate timer to trigger these.